runtime/ramcache/JVMImage.hpp

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
```

Comment:
if you put this file in a new directory you'll have to add a module.xml and a CMakeList.txt file to specify the build instructions (ie create a static/shared lib). Then youll need to update the module.xml/CMakeList in the `runtime/vm` dir to link with your library.

For now, I would suggest keeping all your files in the `runtime/vm` dir, this will simplify things for you

Reply:
This would be nicer in the future. Will put a task for it but won't do it in this pr